birdhouse/components/README.rst

```diff
@@ -153,17 +153,14 @@ Prometheus stack is used:
 Usage
 -----
 
-- Grafana to view metric graphs: http://PAVICS_FQDN:3001/d/pf6xQMWGz/docker-and-system-monitoring
-- Prometheus alert rules: http://PAVICS_FQDN:9090/rules
```

Comment:
I did not add the prometheus stack to the proxy initially so it is not exposed to the internet.  Only someone over VPN can access it because prometheus and alertmanager do not have any authentication.  So I would rather this stays as-is.